config/samples/udg2/udg_external_graphql_internal_rest.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: rest-countries
+spec:
+  name: rest-countries
+  use_keyless: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: https://restcountries.com/v2/alpha/
+    listen_path: /restCountries
+    strip_listen_path: true
+  version_data:
+    default_version: Default
+    not_versioned: true
+    versions:
+      Default:
+        name: Default
+
+
+
+---
+
+
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: udg
+spec:
+  name: Universal Data Graph v2b
```

Comment:
Maybe we can change the names of APIs. Instead of adding the `v2b`, `v2c` suffix, what do you think about naming it `Universal Data Graph v2 with an internal REST API` or something like this?

Reply:
I remember I tried that but it crashed the UI API-screen because the name was too long.